pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
 "tqdm>=4.66.2,<5.0",
 "trl>=0.9.3,<1.0",
 "peft>=0.8.0,<0.13",
+"protobuf==5.28.0",
```

Comment:
This doesn't have to be set on a specific version, we can set the lower bound to 5.28.0 and the upperbound to the next major version 6.0.0. Thus we get automatic updates within 5.x which should not include breaking changes